package.json

```diff
@@ -7,7 +7,9 @@
     "url": "git://github.com/edx/edx-enterprise"
   },
   "dependencies": {
-    "@edx/edx-bootstrap": "^0.3.0"
+    "@edx/brand": "npm:@edx/brand-edx.org@^1.3.0",
+    "@edx/edx-bootstrap": "^0.3.0",
```

Comment:
Is there still a dependency on `@edx/edx-bootstrap` in edx-enterprise? If not, should this be removed from package.json?